docs/antrea-network-policy.md

```diff
@@ -412,6 +426,14 @@ spec:
         ports:
           - protocol: TCP
             port: 8080
+        portRanges:
```

Comment:
we may wanna check to see if the new proposal upstream is accepted. in short, the `except` will go away and instead of a new `portRanges` field, existing `ports` will now include an `endPort` field.